### ISSUE TITLE
Error syntax error, unexpected '->' (T_OBJECT_OPERATOR)

### ISSUE THREAD

**Issue by alanmels** [backdrop-contrib/devel#85]
Looks like with the commit on https://github.com/backdrop-contrib/devel/commit/1e77a6c2a0a004d2c4e8043e27844f1652a168ee several occurrences of `config_get()` got replaced with `config->get()` and hence the error.

Compare the line 1414: https://github.com/backdrop-contrib/devel/blob/6364b3574fa9149a22be8028759b155f83f3bee0/devel.module#L1414

```
$api = config->get('api_url');
```

which should be in fact:

```
$api = config_get('api_url');
```

**Comment by klonos** [comment]
Thanks for catching that @alanmels 👍 ...my bad 😞 

> which should be in fact:
>
> `$api = config_get('api_url');`

Nope, it should actually be `$api = $config->get('api_url');`. See my comments in the PR.

**Comment by alanmels** [comment]
I've got to this page only after replying you on PR. So if the idea was to use `$api = $config->get('api_url');`, then so be it - I'll be submitting new PR for this.

**Comment by alanmels** [comment]
I wanted to create another PR, but the dev branch of the module is not functional in current state, so this needs to go in as soon as possible please.

**Comment by klonos** [comment]
Thanks @alanmels. I've merged this into 1.x 😉
